aca_preview/tests/test_checks.py

```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from proseco import get_aca_catalog
+from proseco.core import StarsTable
+from proseco.tests.test_common import DARK40, STD_INFO
+from aca_preview.preview import ACAReviewTable
+
+
+def test_check_P2():
+    """Test the check of acq P2"""
+    stars = StarsTable.empty()
+    stars.add_fake_constellation(n_stars=3, mag=10.25)
+    aca = get_aca_catalog(**STD_INFO, stars=stars, dark=DARK40)
+    ACAReviewTable.add_review_methods(aca)
+
+    aca.check_acq_p2()
+    assert len(aca.messages) == 1
+    msg = aca.messages[0]
+    assert msg['category'] == 'critical'
+    assert 'less than 2.0 for OR' in msg['text']
```

Comment:
Note that this does not include the actual value of P2 that is reported in the full message, just that some bit of the message matches expectation.  This future-proofs against slight changes due to model updates etc.